assessments/CLI_CIRCLE/seeds/assessments.csv

```diff
@@ -0,0 +1,19 @@
+assessmentIdentifier,assessmentTitle,namespace,assessmentCategoryDescriptor,AcademicSubjectDescriptor
+rln_v1,Rapid Letter Naming,uri://www.cliengage.org/circle/Assessment,uri://ed-fi.org/AssessmentCategoryDescriptor#Early Learning - Language and literacy development,uri://ed-fi.org/AcademicSubjectDescriptor#English Language Arts
+rapid_vocabulary_v1,Rapid Vocabulary,uri://www.cliengage.org/circle/Assessment,uri://ed-fi.org/AssessmentCategoryDescriptor#Early Learning - Language and literacy development,uri://ed-fi.org/AcademicSubjectDescriptor#English Language Arts
+pa_v2,Phonological Awareness,uri://www.cliengage.org/circle/Assessment,uri://ed-fi.org/AssessmentCategoryDescriptor#Early Learning - Language and literacy development,uri://ed-fi.org/AcademicSubjectDescriptor#English Language Arts
```

Comment:
I'm wondering if e.g. `_v2` is relevant/necessary to include in the id. I don't have a super strong opinion, since assessmentTitle is the informative column, but these assessmentIdentifier don't _necessarily_ need to align perfectly with the column names in the file

Reply:
I assumed they had to align to the column names, but if not, I think removing the "_v1", "_v2", and "_2006" would make these titles more legible.